.hermes/conveyor/work-1efc01c2/specs.md

```
@@ -0,0 +1,58 @@
+- No new dependencies, Cargo.lock changes, or build artifacts
+
+## Files Modified
+
+| File | Change |
+|---|---|
+| `crates/perl-lsp-launcher/src/lib.rs` | Add lint enforcement directives (2 lines after line 8); add `#[allow(clippy::print_stderr)]` to `startup_banner` function (line 775) |
+
+## Verification
+
+Run:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)